amaranth/ml/train.py

```diff
@@ -82,25 +83,22 @@ def main():
 
   # Create tokenizer to encode input text
   tokenizer = dict()  # Dictionary mapping each unique word to a unique int
-  tokenizer_cnt = dict()  # Counts appearances of each word in tokenizer
+  tokenizer_cnt = defaultdict(
```

Comment:
defaultdict(int) defaults to 0. https://docs.python.org/3/library/collections.html#defaultdict-examples

Reply:
Oh, even better! I'll adjust

Reply:
Updated in https://github.com/googleinterns/amaranth/pull/35/commits/20b457cc1a4bd755ab253dabffe659f1418f33ab